### DISCUSSION TITLE
Bad Geoserver Calls in 8.0.0-alpha.66

### DISCUSSION THREAD

**Discussion by josecarlosrodriguezcortes** [TerriaJS/terriajs#5296] [Q&A]
Hello.
I'm using the new alpha version and I found that: when I try to get the region mapping using tile vectors I receive always a **404** response with the next message when I try to load a CSV with region mapping (this is an example): `Coverage [minx,miny,maxx,maxy] is [115, 148, 118, 149, 8], index [x,y,z] is [113, 106, 8]` but if I click on the map, after loading the dataset, I receive the tile vector correctly and a **200** response.

I'm using Geoserver to serve my vector tiles and I'm using this URL in the config:
`"server": "https://estadisticas.arte-consultores.com/geoserver/gwc/service/wmts?REQUEST=GetTile&SERVICE=WMTS&VERSION=1.0.0&LAYER=mapastematicos:municipios_20170101_antes2007&STYLE=&TILEMATRIX=EPSG:3857:{z}&TILEMATRIXSET=EPSG:3857&FORMAT=application/vnd.mapbox-vector-tile&TILECOL={x}&TILEROW={y}",`

I also tried to use the TMS service (`https://estadisticas.arte-consultores.com/geoserver/gwc/service/tms/1.0.0/mapastematicos:municipios_20170101_antes2007@EPSG:3857@pbf/{z}/{x}/{y}.pbf`), but the errors were bigger.

I used the `flipY=true` option too in the TMS call but the error persisted. I always have gap between the requested Y and the minY

My data is in **EPSG:4326** but I reprojected it in **EPSG:3857**, though I tried to use **EPSG:4326** directly in the URL and it failed too.

**Comment by tephenavies** [comment]
Hi @josecarlosrodriguezcortes. Were your tiles working in any other version of TerriaJS? 

I think a good way to start debugging region mapping with vector tiles is to use your vector tile endpoint in an `mvt` item. E.g. http://ci.terria.io/next/#clean&https://gist.githubusercontent.com/steve9164/1988a43e34cde90dabfea6842c956523/raw/9591ea779dee51ac49a6cb9710c1b5607798fac6/test-mvt.json (see [catalog](https://gist.githubusercontent.com/steve9164/1988a43e34cde90dabfea6842c956523/raw/9591ea779dee51ac49a6cb9710c1b5607798fac6/test-mvt.json))

**Reply by josecarlosrodriguezcortes** [reply]
Thanks for your advise.
I've found that I can't visualize my vector tiles in Terria but, if I use the same endpoint in QGis, it works without any problem.
